spread.yaml

```diff
@@ -402,13 +402,81 @@ path: /home/gopath/src/github.com/snapcore/snapd
 
 exclude:
     - .git
-    - cmd/snap/snap
-    - cmd/snapd/snapd
-    - cmd/snapctl/snapctl
-    - cmd/snap-exec/snap-exec
-    - cmd/autom4te.cache
-    - "*.o"
+
+    - share
```

Comment:
won't this drop `tests/lib/snaps/test-snapd-layout/usr/share` too? :smiling_imp:

Reply:
💣